workflow-rc2-e2e/tpl/workflow-e2e-pod.yaml

```diff
@@ -17,6 +17,8 @@ spec:
         value: "{{env "GINKGO_NODES" | default 15}}"
       - name: JUNIT
         value: "true"
+      - name: TEST
+        value: "{{env "TEST" | default e2e}}"
```

Comment:
This could technically just be `value: "{{env "TEST"}}"` per https://github.com/smothiki/workflow-e2e/blob/9a55c84acdc224590b225195bb00b6552ac71981/docker-test-integration.sh#L17-L22 right?  (In that link, it looks like the default case is anything other than `TEST='bps'` at this point)